aioxmpp/forms/xso.py

```diff
@@ -22,7 +22,7 @@
 import collections
 import enum
 
-import aioxmpp
+import aioxmpp.stanza
 import aioxmpp.xso as xso
```

Comment:
It is not unrelated. It is there to break an import loop. I don’t know the exact loop off the top of my head, but if you change it back you’ll see the breakage.